docs/guides/custom-transpiler-pass.ipynb

```diff
@@ -83,7 +83,7 @@
     "dag = circuit_to_dag(qc)\n",
     "dag_drawer(dag)\n",
     "```\n",
-    "![](/images/guides/custom-transpiler-pass/DAG.png)\n",
```

Comment:
This image was missing the alt text. Looking at the notebook (https://docs.quantum.ibm.com/guides/custom-transpiler-pass) I think the alt text of the previous image (the circuit inside the "Background: DAG representation" collapsible) refereed to this one, so I moved it here and added an alt text for the circuit